src/pages/dashboard/gcpDashboard/gcpDashboardWidget.tsx

```diff
@@ -7,10 +7,14 @@ import { withTranslation } from 'react-i18next';
 import { connect } from 'react-redux';
 import { createMapStateToProps } from 'store/common';
 import { gcpDashboardActions, gcpDashboardSelectors, GcpDashboardTab } from 'store/dashboard/gcpDashboard';
+import { forecastSelectors } from 'store/forecasts';
 import { reportSelectors } from 'store/reports';
 import { ComputedGcpReportItemsParams } from 'utils/computedReport/getComputedGcpReportItems';
 
+import { awsDashboardActions } from '../../../store/dashboard/awsDashboard';
```

Comment:
Can you please use absolute paths ; for example, 'store/dashboard/awsDashboard'

Reply:
removed this as it was the wrong import